### DISCUSSION TITLE
Domain Name in different language

### DISCUSSION THREAD

**Discussion by garbhagudi** [vercel/community#3458] [Help] [answered]
### Summary

Hi, 

I recently acquired a domain which is not in English. I would like to deploy a project to that domain when I try to add the domain It says "The specified value "www.ಗರ್ಭಗುಡಿ.com" is not a fully qualified domain name." 

Can someone please help me with this. 

Thanks in advance. 

### Steps to Reproduce

Add this domain www.ಗರ್ಭಗುಡಿ.com in the Vercel domain set-up.

**Comment by swarnava** [comment, marked as answer]
Hi,

You can try Punnycode to setup your domain: https://www.punycoder.com/
